src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -16,6 +16,15 @@ typedef enum {
     /* 2 */ IK_TYPE_WHITE
 } EnIkType;
 
+typedef enum {
+    /* 0x0 */ DMG_EFF_HIT,
+    /* 0x4 */ DMG_EFF_FIRE = 0x2,
+    /* 0x0 */ DMG_EFF_ICE,
+    /* 0x4 */ DMG_EFF_LIGHT_SPARKS,
+    /* 0xF */ DMG_EFF_NONE = 0xF,
+
+} EnIkDmgEff;
```

Comment:
```suggestion
typedef enum {
    /* 0x0 */ DMG_EFF_HIT,
    /* 0x4 */ DMG_EFF_FIRE = 0x2,
    /* 0x0 */ DMG_EFF_ICE,
    /* 0x4 */ DMG_EFF_LIGHT_SPARKS,
    /* 0xF */ DMG_EFF_NONE = 0xF,
} EnIkDamageEffect;
```
NONE usually means "no extra effect", while if the attack does absolutely nothing you can say "immune" or similar.

This is also better-placed just above where it's used in the damagetable.